### ISSUE TITLE
Infusion table Recipe does not work.

### ISSUE THREAD

**Issue by Oroton** [paulevsGitch/BetterEnd#181]
Using the template provided I am still unable to get the Infusion table to work.
```
{
    "group": "infusion",
    "input": "minecraft:book",
    "result": {
        "item": "minecraft:diamond_sword"
    },
    "catalysts": {
        "north":      "betterend:enchanted_petal",
        "north_east": "minecraft:lapis_lazuli",
        "east":       "betterend:crystal_shards",
        "south_east": "minecraft:lapis_lazuli",
        "south":      "betterend:crystal_shards",
        "south_west": "minecraft:lapis_lazuli",
        "west":       "betterend:crystal_shards",
        "north_west": "minecraft:lapis_lazuli"
    }
}

```

**Comment by Oroton** [comment]
No just placing them. The infusion table works with the default recipe just not with the data pack one. I thought maybe it was syntax related but I couldn’t work it out

**Comment by paulevsGitch** [comment]
In the next update it will work, I will update wiki later

![image](https://user-images.githubusercontent.com/17707380/121552276-03b32f80-ca19-11eb-9a01-305430b21bb7.png)

**Comment by LTCatt** [comment]
Could you please tell me how long I need to wait for the next release to fix this issue?
My modpack production progress is stuck by this... :(

**Comment by paulevsGitch** [comment]
Probably week or two

**Comment by paulevsGitch** [comment]
Fixed in 0.9.7, I will update wiki with fixed recipe format
